libsignal-service/src/cipher.rs

```diff
@@ -121,7 +141,7 @@ where
                 .await?;
                 let metadata = Metadata {
                     sender: envelope.source_address(),
-                    sender_device: envelope.source_device(),
+                    sender_device: envelope.source_device().into(),
```

Comment:
This is still a `u32`, no?

Reply:
Indeed, I started to change it to `DeviceId` everywhere and reverted because it's somewhat unrelated. This is just a remnant, I just filed https://github.com/whisperfish/libsignal-service-rs/issues/173.